app/controllers/home_page_controller.rb

```diff
@@ -1,8 +1,9 @@
+# frozen_string_literal: true
+
 class HomePageController < ApplicationController
   before_action :authenticate_user!
 
-  def show
-  end
+  def show; end
```

Comment:
@sinpai let's use
```ruby
def show
end
```
style

Reply:
Oh, I gave a power to `rubocop -a` and he went like a tornado through my project